src/azure-cli/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -271,7 +271,7 @@ def list_associated_items_for_policy(client, resource_group_name, vault_name, na
 
 def list_protectable_items(cmd, client, resource_group_name, vault_name, workload_type,
                            backup_management_type="AzureWorkload", container_name=None, protectable_item_type=None,
-                           server_name=None):
+                           server_name=None, instance_name=None):
```

Comment:
are we creating a new parameter for list protectable items command ?

Reply:
It was adding one, I've resolved that with the latest update.